tests/ui/should-panic-attribute/expected-panics/test.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-
+//
+// compile-flags: -Copt-level=1
```

Comment:
Why was this added?

Reply:
Because the expected file checks the number of existing assertions which went up from 2 to 6. However, those are dead assumption.